### PR TITLE
Adjusted the extension to properly use our scoring algorithm.

### LINKED ISSUE #61: add sighted keyboard to algo

### PR DESCRIPTION
**Pull request by dannyT2018**
Implemented the Ha11y Chrome Extension to utilize our current scoring algorithm.

### CODE REVIEW

File: api/index.js [hunk 1]
```diff
@@ -15,6 +13,53 @@ app.use(bodyParser.json());
...
+    }
+    if (disabilities.charAt(attentionDeficitIndex+1) == 't'){
+        isAttentionDeficit = true;
+    }
+    if (disabilities.charAt(sightedKeyboardUsersIndex+1) == 't'){
+        isSightedKeyboardUsers = true;
+    }
+    if (disabilities.charAt(lowVisionIndex+1) == 't'){
+        isLowVision = true;
+    }
```
Comment (BracciaCJ) [inline api/index.js]: This will work, however, we may want to dicsuss more scalable options

File: api/index.js [hunk 2]
```diff
@@ -15,6 +13,53 @@ app.use(bodyParser.json());
...
+    let motorIndex = disabilities.search("m");
+    let cognitiveIndex = disabilities.search("g");
+    let attentionDeficitIndex = disabilities.search("a");
+    let sightedKeyboardUsersIndex = disabilities.search("s");
+    let lowVisionIndex = disabilities.search("l");
+        
+    let isBlind;
+    let isDeaf;
+    let isColorBlind;
+    let isMotor;
```
Comment (BracciaCJ) [inline api/index.js]: This need to be set to false by default

File: api/index.js [hunk 3]
```diff
@@ -34,18 +79,51 @@ app.get('/', async function(req, res) {
...
+                            maxDeduction = row['sighted keyboard users']
+                        }
+                    }
+                    if (row['low vision'] && isLowVision){
+                        if (row['low vision'] <= maxDeduction){
+                            maxDeduction = row['low vision']
+                        }
+                    }
+
+                    jsonObj['totalScore'] += parseInt(maxDeduction);
```
Comment (BracciaCJ) [inline api/index.js]: Should we consider switching the numbers in the csv to be pos?

**Review by BracciaCJ — CHANGES_REQUESTED** [review]
Only real concern I see is 27-34 should be set to false.

**Comment by dannyT2018** [comment]
Close #61 #50 #48 #46 #45

**Review by emma-hubka — APPROVED** [review]

**Pull request merged**